samples/bluetooth/direction_finding_connectionless_rx/README.rst

```diff
@@ -16,7 +16,7 @@ The sample supports the following development kits:
 
 .. table-from-rows:: /includes/sample_board_rows.txt
    :header: heading
-   :rows: nrf52833dk_nrf52833
+   :rows: nrf52833dk_nrf52833, nrf5340dk_nrf5340_cpuapp_and_cpuappns
```

Comment:
Can we use `nrf5340dk_nrf5340_cpuapp_and_cpuappns`, or should we split this into 2 boards?

Reply:
`nrf5340dk_nrf5340_cpuapp_and_cpuappns` is used by other BLE samples in nrf-sdk, so I assume that this it the correct one.